packages/create-blade/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "create-blade",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.9.3",
```

To ensure that the release CI works correctly.